README.md

```diff
@@ -82,7 +82,7 @@ pip install sparsezoo
 
 ## Quick Tour
 
-Each model in the SparseZoo has a specific stub that identifies it. The stubs are made up of the following structure:
+Each model in SparseZoo has a specific stub that identifies it. The stubs are made up of the following structure:
```

Comment:
We are removing the sparsezoo stub overview here in favor of using what is at models.md - could we apply these edits to that?